client/ayon_photoshop/plugins/publish/collect_review.py

```diff
@@ -24,3 +24,11 @@ def process(self, context):
             if (creator_attributes.get("mark_for_review") and
                     "review" not in instance.data["families"]):
                 instance.data["families"].append("review")
+
+            if "tags" not in instance.data:
```

Comment:
`tags` on instance?

Reply:
I see tags on instance only:
- https://github.com/ynput/ayon-hiero/blob/5a4969c917d80bd24191b73a6004a7bd9d934592/client/ayon_hiero/plugins/publish/collect_tag_tasks.py#L15
- https://github.com/ynput/ayon-hiero/blob/f930b23c78e17efbd7e037453a46f3654fb15107/client/ayon_hiero/plugins/publish_old_workflow/collect_tag_comments.py#L17

So seems to be only there for `hiero`